gc/base/standard/ConfigurationGenerational.hpp

```diff
@@ -61,6 +61,7 @@ class MM_ConfigurationGenerational : public MM_ConfigurationStandard
 	};
 	
 protected:
+	virtual bool initialize(MM_EnvironmentBase* env);
```

Comment:
I know our code is inconsistent in this regard in the existing code, but I prefer that init method is not virtual.